ckan/lib/helpers.py

```diff
@@ -1140,37 +1136,13 @@ def has_more_facets(facet: str,
             facets.append(dict(active=False, **facet_item))
         elif not exclude_active:
             facets.append(dict(active=True, **facet_item))
-    if getattr(c, 'search_facets_limits', None) and limit is None:
-        limit = c.search_facets_limits.get(facet)
+    if getattr(g, 'search_facets_limits', None) and limit is None:
+        limit = g.search_facets_limits.get(facet)
     if limit is not None and len(facets) > limit:
         return True
     return False
 
 
-@core_helper
-def unselected_facet_items(
```

Comment:
this helper can be used in extensions, maybe we should mark first as deprecated and remove it in next release ?

Reply:
As discussed in the Tech Team meeting, I've added a change file for this change.